Sources/App/Views/Sections/Editor/Detail/EditorDetail.swift

```diff
@@ -9,6 +9,9 @@ struct EditorDetail: View {
 
   // MARK: - Stored Properties
 
+  /// The current color scheme of the app.
+  @Environment(\.colorScheme) var colorScheme: ColorScheme
```

Comment:
Check if this is really needed

Reply:
Unfortunately yes. I was looking into a way for centralizing it. 
I tried to move this into an `extension` but it didn't work as that way because `colorScheme` is not injected in the `View`.